src/middleware/gpsd/groups.h

```diff
@@ -0,0 +1,44 @@
+// Copyright 2020:
+//   GobySoft, LLC (2013-)
+//   Community contributors (see AUTHORS file)
+// File authors:
+//   Shawn Dooley <shawn@shawndooley.net>
+//
+//
+// This file is part of the Goby Underwater Autonomy Project Libraries
+// ("The Goby Libraries").
+//
+// The Goby Libraries are free software: you can redistribute them and/or modify
+// them under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 2.1 of the License, or
+// (at your option) any later version.
+//
+// The Goby Libraries are distributed in the hope that they will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with Goby.  If not, see <http://www.gnu.org/licenses/>.
+
+#pragma once
+
+#include "goby/middleware/group.h"
+
+namespace goby
+{
+namespace middleware
+{
+namespace groups
+{
+namespace gpsd 
+{
+constexpr goby::middleware::Group tpv("goby::apps::zeromq::gpsd::tpv");
```

Comment:
Change to "goby::apps::middleware::gpsd::tpv" to match namespace? (I really should probably make a macro for this at some point). Similar for next two lines.